workflow/signal/signal.go

```diff
@@ -49,6 +50,8 @@ func ExecPodContainerAndGetOutput(ctx context.Context, restConfig *rest.Config,
 	if err != nil {
 		return err
 	}
+	ctx, cancel := context.WithTimeout(ctx, 2*time.Minute)
+	defer cancel()
```

Comment:
@agilgur5 I modified it in this way. Thanks.

Reply:
I thought about it and it really doesn't need to be configurable.

Reply:
>  you didn't answer the above; do you know why it works that way?

hi @agilgur5 , Colud i ask which question are you talking about ? That message is too lang, i don't understand which one

Reply:
[This one](https://github.com/argoproj/argo-workflows/pull/13012#discussion_r1594598882)